### PR TITLE
Implement action options validation (closes #420)

### LINKED ISSUE #420: Implement action options validation
- [x] Move client options to single file
  - [x] Implement assignment from object in ctor and test it
  - [x] Implement error for the incorrect option type
  - [x] Write formatter test for this error
  - [x] Implement option validation and write tests for it

### PR DESCRIPTION
**Pull request by inikulin**
\cc @AlexanderMoskovkin @miherlosev

### CODE REVIEW

**Comment by inikulin** [comment]
@AlexanderMoskovkin Note: we now initialize options with the object passed to the constructor. Also, we will not need server-side classe, as we discussed previously.

File: src/client/runner/api/actions.js [hunk 1]
```diff
@@ -751,16 +731,13 @@ export function hover (what, options) {
...
 
-                hoverOptions.offsetX = offsetX;
-                hoverOptions.offsetY = offsetY;
-
-                hoverOptions.modifiers.ctrl  = options.ctrl;
-                hoverOptions.modifiers.alt   = options.alt;
-                hoverOptions.modifiers.shift = options.shift;
-                hoverOptions.modifiers.meta  = options.meta;
+                var hoverOptions = new MouseOptions({
+                    offsetX,
```
Comment (miherlosev) [inline src/client/runner/api/actions.js]: Maybe do we need to rename it from
` var hoverOptions = new MouseOptions({`
to
` var mouseOptions = new MouseOptions({`

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by miherlosev** [comment]
Lgtm

**Pull request merged**